src/libraries/System.Diagnostics.PerformanceCounter/tests/PerformanceCounterCategoryTests.cs

```diff
@@ -232,7 +232,7 @@ public static void PerformanceCounterCategory_GetCounters()
             PerformanceCounter[] counters = pcc.GetCounters();
```

Comment:
One line 212 in `PerformanceCounterCategory_DeleteCategory` please update it to call `Helpers.DeleteCategory` instead of `PerformanceCounterCategory.Delete`